bin/demo.py

```diff
@@ -89,40 +103,43 @@ def verify_jenkins(jenkins_url):
 
 def install_marathon_lb(marathon_lb_url):
     log("installing marathon-lb")
-    install_package_and_wait('marathon-lb')
-    assert package_installed('marathon-lb'), log_and_exit('!! package failed to install')
-    log("waiting for marathon-lb service to come up at '{}'".format(marathon_lb_url))
-    end_time = time.time() + 300
-    while time.time() < end_time:
-        if verify_marathon_lb(marathon_lb_url):
-            break
-        time.sleep(1)
+    with stdchannel_redirected(sys.stdout, os.devnull):
+        run_dcos_command('marathon app add conf/get_sa.json')
+        end_time = time.time() + 300
+        while time.time() < end_time:
+            if get_marathon_task('saread'):
+                break
+            time.sleep(1)
+        log("retrieving service account JSON")
+        satoken = run_dcos_command("task log --lines=1 saread")[0]
+        run_dcos_command('marathon app remove saread')
+    post_url = "{}secrets/v1/secret/default/marathon-lb".format(dcos_url)
```

Comment:
Can we add a graceful catch for if this secret already exists? I just ran this twice on the same cluster and got this exception:
```
➜  cd-demo git:(sunil-demo2) ✗ python3 bin/demo.py pipeline --password=$PASSWORD --branch=sunil-demo2 http://sunil-f4j-publicsl-9adnwfg46s3s-1406902471.us-west-2.elb.amazonaws.com/ http://sunil-f4j-elasticl-1hx6ny1cqwidv-1482094555.us-west-2.elb.amazonaws.com/
[demo]: couldn't find marathon-lb running at 'http://sunil-f4j-publicsl-9adnwfg46s3s-1406902471.us-west-2.elb.amazonaws.com/'
[demo]: installing marathon-lb
Traceback (most recent call last):
  File "bin/demo.py", line 272, in <module>
    install_marathon_lb(elb_url)
  File "bin/demo.py", line 119, in install_marathon_lb
    r = http.put(post_url, headers=headers, data=data)
  File "/usr/local/lib/python3.5/site-packages/dcos/http.py", line 297, in put
    return request('put', url, data=data, **kwargs)
  File "/usr/local/lib/python3.5/site-packages/dcos/http.py", line 235, in request
    raise DCOSHTTPException(response)
dcos.errors.DCOSHTTPException: Error while fetching [http://sunil-f4j-elasticl-1hx6ny1cqwidv-1482094555.us-west-2.elb.amazonaws.com/secrets/v1/secret/default/marathon-lb]: HTTP 409: Conflict
```